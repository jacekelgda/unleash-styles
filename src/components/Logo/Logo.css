.Logo {
    display: flex;
    text-decoration: none;
}

.LogoImg {
    display: inline-block;
    width: 36px;
    height: 46px;
}

.LogoText {
    display: none;
}
@media screen and (min-width: 1024px) {
    .LogoText {
        display: inline-block;
        margin-left: 13px;
        font: 800 22px/46px 'Montserrat', sans-serif;
        text-transform: uppercase;
        color: #0c0c0c;
    }
}