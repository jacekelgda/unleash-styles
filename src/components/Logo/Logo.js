/** @flow */
import React from 'react'
import styles from './Logo.css'

const logoImg = require('../../assets/logo.png')

/**
 * Unleash logo component
 */
const Logo = () => (
  <div>
    <a href={'#'} className={styles.Logo}>
      <img src={logoImg} alt={'Unleash'} className={styles.LogoImg} />
      <span className={styles.LogoText}>Unleash</span>
    </a>
  </div>
)

export default Logo
