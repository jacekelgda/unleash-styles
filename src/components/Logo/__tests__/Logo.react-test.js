/* global test, expect */
import React from 'react'
import { shallow } from 'enzyme'

import Logo from '../index'

test('Logo renders correctly', () => {
  const wrapper = shallow(<Logo />)
  expect(wrapper).toMatchSnapshot()
})
