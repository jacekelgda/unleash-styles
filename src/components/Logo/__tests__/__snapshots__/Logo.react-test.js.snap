// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<div>
  <a
    className="Logo"
    href="#"
  >
    <img
      alt="Unleash"
      className="LogoImg"
      src="test-file-stub"
    />
    <span
      className="LogoText"
    >
      Unleash
    </span>
  </a>
</div>
`;
